### ISSUE TITLE
[BUG] cuspatial._lib.soa_readers example out of date with API?

### ISSUE THREAD

**Issue by randerzander** [rapidsai/cuspatial#47]
I'm trying to follow [this example](https://github.com/rapidsai/cuspatial/blob/branch-0.10/python/cuspatial/demos/pip_verify_shapely_locust.py#L26).

```
import cuspatial._lib.soa_readers as readers
import cuspatial._lib.spatial as gis

fpos, rpos, plyx, plyy = readers.cpp_read_ply_soa(fn)
```
Result:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-19-8c68f35633c5> in <module>
      3 
      4 
----> 5 fpos, rpos, plyx, plyy = readers.cpp_read_ply_soa(fn)

AttributeError: module 'cuspatial._lib.soa_readers' has no attribute 'cpp_read_ply_soa'
```

**Comment by randerzander** [comment]
I tried `cpp_read_polygon_soa(soa_file_name)` instead and get a different error:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-22-a89fda1f9a7e> in <module>
      2 import cuspatial._lib.spatial as gis
      3 
----> 4 fpos, rpos, plyx, plyy = readers.cpp_read_polygon_soa(fn)

cuspatial/_lib/soa_readers.pyx in cuspatial._lib.soa_readers.cpp_read_polygon_soa()

cuspatial/_lib/soa_readers.pyx in cuspatial._lib.soa_readers.cpp_read_polygon_soa()

RuntimeError: cuDF failure at: /rapids/cuspatial/cpp/src/utility/utility.hpp:192: numbers of groups/features/rings/vertices must be positive
```

This is against a file that imports ok with `shapefile` as in the [shapely_locust](https://github.com/rapidsai/cuspatial/blob/branch-0.10/python/cuspatial/demos/pip_test_shapely_locust.py) demo:
```
plyreader = shapefile.Reader(fn)
polygon = plyreader.shapes()
plys = []
for shape in polygon:
    plys.append(Polygon(shape.points))
```

**Comment by thomcom** [comment]
Sorry, Randy. This PR has the updated demos:
https://github.com/rapidsai/cuspatial/pull/44

**Comment by thomcom** [comment]
Fixed by #44
